CONTRIBUTING.md

```diff
@@ -167,22 +176,60 @@ func delete(w io.Writer, name string) error {
 }
 ```
 
+## Function arguments for quickstarts
```

Comment:
Change the "Function arguments" section to be "Function arguments for snippets"?

Reply:
Since quickstarts can't have function arguments in the same way that other snippets can, my intent here is to specifically highlight how to pass args into a quickstart (notably, via flags)

Reply:
Sorry, I didn't mean to change this section. Just rename the other section to make it clear there is one for snippets and one for quickstarts.